docs/docs/event-sources/http/README.md

```diff
@@ -2,10 +2,13 @@
 
 This tutorial will walk you through how to:
 
-1. [Install the Pipedream CLI](#install-the-pipedream-cli)
-2. [Link the CLI to your Pipedream account](#sign-up-for-pipedream)
-3. [Create an HTTP source](#create-an-http-source)
-4. [Consume events from that source](#consume-events-from-your-http-source)
+- [Creating an HTTP source](#creating-an-http-source)
```

Comment:
Same ToC issue here, I'd maybe just revert the change? I think we originally had the numbers there for clarity since it was a set of steps the user was supposed to follow.

Reply:
done